src/test/resources/application.conf

```diff
@@ -17,9 +17,17 @@ environment: local
 local {
   services {
     host: "http://localhost"
-    service-name-frontend {
-      port: 1234
-      productionRoute: "/path"
+    auth {
+      port: 8585
+      productionRoute: ""
```

Comment:
What happens if you omit the empty keys?

Reply:
Its been used in code to form the full URL so